From 69e289da37f19ed6d80eb7d12a62adf2c76bc332 Mon Sep 17 00:00:00 2001
From: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Date: Thu, 21 Sep 2023 10:14:19 +0200
Subject: [PATCH 367/954] refactor(st-clock): rename USB3DRD

Rename USB3DRD in USB3DR.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 2381493f9ae46798581d445fc26805cfa2036185 refactor(st-clock): rename USB3DRD]
Change-Id: Idbfcbae472f3b16d464c5e9522adeb4928eaf66d
Signed-off-by: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/328803
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 drivers/st/clk/stm32mp2_clk.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/st/clk/stm32mp2_clk.c b/drivers/st/clk/stm32mp2_clk.c
index 099b962c66..8c8f714432 100644
--- a/drivers/st/clk/stm32mp2_clk.c
+++ b/drivers/st/clk/stm32mp2_clk.c
@@ -247,7 +247,7 @@ enum clock {
 	_CK_USB2OHCI,
 	_CK_USB2PHY1,
 	_CK_USB2PHY2,
-	_CK_USB3DRD,
+	_CK_USB3DR,
 	_CK_USB3PCIEPHY,
 	_CK_USBTC,
 
@@ -364,7 +364,7 @@ enum enum_gate_cfg {
 	GATE_USB2OHCI,
 	GATE_USB2EHCI,
 
-	GATE_USB3DRD,
+	GATE_USB3DR,
 
 	GATE_BSEC,
 	GATE_IWDG1,
@@ -456,7 +456,7 @@ static const struct gate_cfg gates_mp25[LAST_GATE] = {
 	/* WARNING 2 CLOCKS FOR ONE GATE */
 	GATE_CFG(GATE_USB2OHCI,		RCC_USB2CFGR,		1,	0),
 	GATE_CFG(GATE_USB2EHCI,		RCC_USB2CFGR,		1,	0),
-	GATE_CFG(GATE_USB3DRD,		RCC_USB3DRDCFGR,	1,	0),
+	GATE_CFG(GATE_USB3DR,		RCC_USB3DRCFGR,		1,	0),
 	GATE_CFG(GATE_BSEC,		RCC_BSECCFGR,		1,	0),
 	GATE_CFG(GATE_IWDG1,		RCC_IWDG1CFGR,		1,	0),
 	GATE_CFG(GATE_IWDG2,		RCC_IWDG2CFGR,		1,	0),
@@ -1204,7 +1204,7 @@ static const struct clk_stm32 stm32mp25_clk[CK_LAST] = {
 	STM32_GATE(_CK_USB2OHCI, CK_BUS_USB2OHCI, _CK_ICN_HSL, 0, GATE_USB2OHCI),
 	STM32_GATE(_CK_USB2EHCI, CK_BUS_USB2EHCI, _CK_ICN_HSL, 0, GATE_USB2EHCI),
 
-	STM32_GATE(_CK_USB3DRD, CK_BUS_USB3DRD, _CK_ICN_HSL, 0, GATE_USB3DRD),
+	STM32_GATE(_CK_USB3DR, CK_BUS_USB3DR, _CK_ICN_HSL, 0, GATE_USB3DR),
 
 	STM32_GATE(_CK_BSEC, CK_BUS_BSEC, _CK_ICN_APB3, 0, GATE_BSEC),
 	STM32_GATE(_CK_IWDG1, CK_BUS_IWDG1, _CK_ICN_APB3, 0, GATE_IWDG1),
-- 
2.53.0

